frontend/src/apis/LogoutAPI.js

```diff
@@ -0,0 +1,19 @@
+import API from './Base';
+
+const LogoutAPI = () => {
+  const logout = () => {
+    API.post('/logout').then((res) => {
+      if (res.data.status === 200) {
+        localStorage.removeItem('auth_token');
+        localStorage.removeItem('auth_name');
```

Comment:
please store it on cookies instead of localstorage. Please search for cookies package. Cookies are much smaller memory than localstorage.